<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="CVImageWidget" name="wgtImg" native="true"/>
    </item>
    <item>
     <widget class="QListView" name="cmdListView">
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wgtTools" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="FileWidget" name="wgtFile" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuModification">
    <property name="title">
     <string>Modification</string>
    </property>
    <addaction name="actionLinear_Scaling"/>
    <addaction name="actionGamma"/>
    <addaction name="actionEqualDensity"/>
    <addaction name="actionHistogram_Equalization"/>
    <addaction name="actionCanny"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <widget class="QMenu" name="menuDifference">
     <property name="title">
      <string>Difference</string>
     </property>
     <addaction name="actionX"/>
     <addaction name="actionY"/>
     <addaction name="actionBoth"/>
    </widget>
    <widget class="QMenu" name="menuCompass">
     <property name="title">
      <string>Compass</string>
     </property>
     <addaction name="actionNorth"/>
     <addaction name="actionNorthEast"/>
     <addaction name="actionEast"/>
     <addaction name="actionSouthEast"/>
     <addaction name="actionSouth"/>
     <addaction name="actionSouthWest"/>
     <addaction name="actionWest"/>
     <addaction name="actionNorthWest"/>
    </widget>
    <addaction name="actionGau"/>
    <addaction name="actionSobel_X"/>
    <addaction name="actionSobel_Y"/>
    <addaction name="actionLaplace"/>
    <addaction name="menuDifference"/>
    <addaction name="menuCompass"/>
   </widget>
   <widget class="QMenu" name="menuMeasure">
    <property name="title">
     <string>Measure</string>
    </property>
    <addaction name="actionNumbers"/>
    <addaction name="actionGreyvalue_Matrix"/>
    <addaction name="actionHistogramm"/>
   </widget>
   <widget class="QMenu" name="menuMorphology">
    <property name="title">
     <string>Morphology</string>
    </property>
    <addaction name="actionDilatation"/>
    <addaction name="actionErosion"/>
    <addaction name="actionMedian"/>
   </widget>
   <addaction name="menuMeasure"/>
   <addaction name="menuModification"/>
   <addaction name="menuFilter"/>
   <addaction name="menuMorphology"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLinear_Scaling">
   <property name="text">
    <string>linear Scaling</string>
   </property>
  </action>
  <action name="actionGamma">
   <property name="text">
    <string>Gamma</string>
   </property>
  </action>
  <action name="actionEqualDensity">
   <property name="text">
    <string>equal Density</string>
   </property>
  </action>
  <action name="actionGau">
   <property name="text">
    <string>Gau√ü</string>
   </property>
  </action>
  <action name="actionSobel_X">
   <property name="text">
    <string>Sobel X</string>
   </property>
  </action>
  <action name="actionSobel_Y">
   <property name="text">
    <string>Sobel Y</string>
   </property>
  </action>
  <action name="actionNumbers">
   <property name="text">
    <string>Numbers</string>
   </property>
  </action>
  <action name="actionGreyvalue_Matrix">
   <property name="text">
    <string>Greyvalue Matrix</string>
   </property>
  </action>
  <action name="actionHistogramm">
   <property name="text">
    <string>Histogramm</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionLaplace">
   <property name="text">
    <string>Laplace</string>
   </property>
  </action>
  <action name="actionX">
   <property name="text">
    <string>X</string>
   </property>
  </action>
  <action name="actionY">
   <property name="text">
    <string>Y</string>
   </property>
  </action>
  <action name="actionBoth">
   <property name="text">
    <string>Both</string>
   </property>
  </action>
  <action name="actionNorth">
   <property name="text">
    <string>North</string>
   </property>
  </action>
  <action name="actionNorthEast">
   <property name="text">
    <string>NorthEast</string>
   </property>
  </action>
  <action name="actionSouthEast">
   <property name="text">
    <string>Southeast</string>
   </property>
  </action>
  <action name="actionSouth">
   <property name="text">
    <string>South</string>
   </property>
  </action>
  <action name="actionSouthWest">
   <property name="text">
    <string>SouthWest</string>
   </property>
  </action>
  <action name="actionWest">
   <property name="text">
    <string>West</string>
   </property>
  </action>
  <action name="actionNorthWest">
   <property name="text">
    <string>NorthWest</string>
   </property>
  </action>
  <action name="actionEast">
   <property name="text">
    <string>East</string>
   </property>
  </action>
  <action name="actionDilatation">
   <property name="text">
    <string>Dilatation</string>
   </property>
  </action>
  <action name="actionErosion">
   <property name="text">
    <string>Erosion</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>Median</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CVImageWidget</class>
   <extends>QWidget</extends>
   <header>cvimagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileWidget</class>
   <extends>QWidget</extends>
   <header>filewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
